hub/api/tests/test_api.py

```diff
@@ -0,0 +1,64 @@
+import os, shutil
+import numpy as np
+from hub.api.dataset import Dataset
+from hub.core.storage import LocalProvider
+from hub.core.chunk_engine.read import read_dataset_meta, read_tensor_meta
+from hub.core.tests.common import parametrize_all_dataset_storages
+
+import pytest
+
+
+def test_create_empty_local_dataset():
+    ds = Dataset("/tmp/hub-test/empty")
+    assert type(ds.provider) == LocalProvider
+    assert read_dataset_meta(ds.provider) == {"tensors": []}
+
+
+def test_persist_local(local_storage):
```

Comment:
you could probably generalize this test to work for `local`/`s3`. if you want to be even more general, i recommend creating another set of `@parametrize` decorators like `parametrize_all_persistent_storages`.

Reply:
Not really. Part of this test is passing the same _path_, not just the same _provider_. I'm gonna leave this as-is for now, it's too much effort for too little gain to generalize this.